idutils/__init__.py

```diff
@@ -239,6 +292,17 @@
 genome_regexp = re.compile(r"GC[AF]_\d+\.\d+$")
 """GenBank or RefSeq genome assembly accession."""
 
+geo_regexp = re.compile(r"G(PL|SM|SE|DS)\d+$")
+"""Gene Expression Omnibus (GEO) accession."""
```

Comment:
As sometimes done in other docstrings for the regular expression definitions, could you also include a link or reference (if there is one) pointing to the definition/specification of the identifiers?